components/NewRegistrationForm/NewRegistrationForm.js

```diff
@@ -0,0 +1,35 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { login } from 'common/utils/auth-utils';
+import MultiStepForm from 'components/Form/MultiStepForm';
+import { Steps, StepsInitialValues } from './Steps';
+
+class NewRegistrationForm extends Component {
+  static propTypes = {
+    initialValues: PropTypes.objectOf(PropTypes.string),
+  };
+
+  static defaultProps = {
+    initialValues: {
+      ...StepsInitialValues,
+    },
+  };
+
```

Comment:
Bring the OrderedSteps array into this file as a method

Use insertIf() utility on Military step against a state variable

Set state based upon specific field value (grab from stepRender’s props)